<?xml version="1.0"?>
<launch>	

    <node name="start_system_node" pkg="agv_navigation" type="start_system_node.py"/>

</launch>